src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm.service.ts

```diff
@@ -71,20 +71,8 @@ export class GermplasmService {
             { params, observe: 'response' });
     }
 
-    getGermplasmAttributes(codes: string[]): Observable<Attribute[]> {
-        const url = SERVER_API_URL + `crops/${this.context.cropName}/germplasm/attributes` +
-            '?programUUID=' + this.context.programUUID + '&codes=' + codes;
-        return this.http.get<Attribute[]>(url);
-    }
-
-    getGermplasmAttributesByType(attributeType: string): Observable<Attribute[]> {
-        const url = SERVER_API_URL + `crops/${this.context.cropName}/germplasm/attributes` +
-            '?programUUID=' + this.context.programUUID + '&type=' + attributeType;
-        return this.http.get<Attribute[]>(url);
-    }
-
-    getGermplasmAttributesByGidAndType(gid: number, type: string): Observable<GermplasmAttribute[]> {
-        const url = SERVER_API_URL + `crops/${this.context.cropName}/germplasm/${gid}/attributes?type=${type}`;
+    getGermplasmAttributesByGidAndType(gid: number, variableTypeId: number): Observable<GermplasmAttribute[]> {
```

Comment:
Can `variableTypeId `here be declard as `VariableType` (suggested to be renamed from VariableTypeEnum currently)?